backend/director.py

```diff
@@ -1,3 +1,18 @@
-# TODO: Add basic director implementation
-def question():
-    return "Hello World!"
+from utils.llm import call_model
+
+prompt= """
+You are a Chat Bot called InferGPT.
+
+You're abilities include:
+
+- Setting and tracking Goals
+- Remembering conversation information from previous sessions and using it in future discussions
+
+Introduce yourself if asked.
+"""
```

Comment:
Does this prompt cover too much at this point as some of that functionality is not included? Also can set this as a system prompt instead of adding to the user prompt

```python
messages=[
            {"role": "system", "content": system_message},
            {"role": "user", "content": user_prompt}
        ],
```

Reply:
Adjusted to separate system and user prompts